npm-shrinkwrap.json

```diff
@@ -2210,6 +2210,15 @@
         "node": ">=0.12"
       }
     },
+    "node_modules/deep-extend": {
```

Comment:
Need to remove this changes, since it was changed when I installed the node-wifi dependency